CONTRIBUTING.rst

```diff
@@ -124,11 +122,13 @@ To set up ScanCode for local development:
 3. Create a branch for local development::
 
     git checkout -b name-of-your-bugfix-or-feature
+    
```

Comment:
This should have a point number, adjust others accordingly. 

Also something like `check out the contributing to developement documentation, as it contains ...` may be better for this?